src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -66,7 +66,9 @@ public class SliderSlidingFinishedEventArgs : EventArgs
     /// <since_tizen> 6 </since_tizen>
     public partial class Slider : Control
     {
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        #region Fields
+
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
We may not open bindable properties. Please remove wrong comment.

Reply:
I've just changed the sentences from 'tizen_6.0' to 'later'. Then, can I leave just [EditorBrowsable] tags?